fastkaggle/core.py

```diff
@@ -1,11 +1,13 @@
 # AUTOGENERATED! DO NOT EDIT! File to edit: ../00_core.ipynb.
 
 # %% auto 0
-__all__ = ['iskaggle', 'import_kaggle', 'setup_comp', 'nb_meta', 'push_notebook']
+__all__ = ['iskaggle', 'import_kaggle', 'setup_comp', 'nb_meta', 'push_notebook', 'mk_dataset', 'get_dataset', 'get_pip_library',
+           'push_dataset']
 
 # %% ../00_core.ipynb 3
-import os,json
+import os,json,subprocess
 from fastcore.utils import *
+from kaggle import api
```

Comment:
Use `import_kaggle()` to import the api, and only do it when needed, since otherwise it breaks when run on Kaggle.

Reply:
I have adjusted this in the notebook and also made that change in a couple already existing functions.